api/src/modules/projects/projects.service.ts

```diff
@@ -300,13 +338,19 @@ export class ProjectsService extends AppBaseService<
           countryCode: updateProjectDto.countryCode,
         },
       });
+    const { costOutput, breakEvenCostOutput } = costs;
+
+    const costsToUse =
```

Comment:
The Update is tricky:

Since for each input there should be 2 rows created (Maket price, and Opex breakeven, as long as it is found), how do we work with the update?

1. Does it make sense to allow the user to update only 1 row (either Market price OR Opex breakeven), when conceptually both rows are the output for the same project). Right now I am doing that, but I doubt this make sense

2. It can also happen that updating a project for the opex breakeven price, the new algorithm does not converge and there is no actually an output for this. How do we inform the user? Should we just keep the old value, and render a notification?